AnagramTS_/src/anagram.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { Anagram } from "./anagram";
+
+describe("testing anagram", () => {
+  test('inputed word "melon" should matched 4 words ', () => {
+    const melonToCheck = new Anagram("melon");
```

Comment:
it is better to organize tests following the AAA convention. in small tests it might seem pointless but the more complex the tests become the more benefits there will be:
//arrange: const melonToCheck = new Anagram("melon");
//act: const result = melonToCheck.checkIsFirstWordAnagram();
//assert" expect(result.length).toBe(4);